app/models/version.js

```diff
@@ -85,7 +88,11 @@ export default class Version extends Model {
     }
 
     let { crate, semver, releaseTrack } = this;
-    let { versions } = crate;
+    let { versions, releaseTrackSet } = crate;
+    if (releaseTrackSet !== undefined) {
```

Comment:
unless I'm misreading the above code, `releaseTrackSet` should never be `undefined` and will always at least be an empty `Set`, right?

Reply:
Similar to the previous reason, this additional check can be seen as a defensive programming measure. If all goes well, it will indeed behave as you described.

Reply:
same here, I think. `return this.crate.releaseTrackSet.has(this.id)` should probably be sufficient. worst case, it will return `false` and not display the right styles, but it won't break the app.